Update 01-deep-insight-into-open-source-databases.mdx (#228)
pingcap/ossinsight: docs/database/01-deep-insight-into-open-source-databases.mdx

## Database user feedback 

> ### Which databases have the widest feedback source?

Reviewer: ```suggestion
### Which databases have the widest feedback sources?
```

> ### Feedback trend in the past ten years

Reviewer: ```suggestion
### Which databases have the widest feedback source?
```

> The chart below displays the number of issues creators of leading open source databases respectively each year and their growth trend during the past ten years.

Reviewer: ```suggestion
The chart below displays the number of issue creators of leading open source databases each year and their growth trend during the past ten years.
```
